tests/testthat/test-as_tibble.R

```diff
@@ -0,0 +1,193 @@
+t1 <- trial |>
+  tbl_summary()
+
+t2 <- glm(response ~ age + grade, trial, family = binomial(link = "logit")) |>
+  tbl_regression(exponentiate = TRUE)
+
+t3 <- tbl_uvregression(
+  trial |> dplyr::select(response, age, grade),
+  method = glm,
+  y = response,
+  method.args = list(family = binomial),
+  exponentiate = TRUE
+)
+
+test_that("as_tibble works with standard use", {
+  expect_silent(res <- as_tibble(t1))
+  expect_silent(as_tibble(t1, return_calls = TRUE))
+  expect_snapshot(res |> as.data.frame())
+
+  expect_silent(res <- as_tibble(t2))
+  expect_snapshot(res |> as.data.frame())
+
+  expect_silent(res <- as_tibble(t3))
+  expect_snapshot(res |> as.data.frame())
+})
+
+test_that("as_tibble(col_labels=) works", {
+  expect_equal(
+    as_tibble(t2, col_labels = FALSE) |> names(),
+    c("label", "estimate", "conf.low", "p.value")
+  )
+
+  expect_equal(
+    as_tibble(t2) |> names(),
+    c("**Characteristic**", "**OR**", "**95% CI**", "**p-value**")
+  )
+})
+
+t1 <- t1 |>
+  bold_labels() |>
+  italicize_levels()
+
+t2 <- t2 |>
+  bold_labels() |>
+  italicize_levels()
+
+t3 <- t3 |>
+  bold_levels() |>
+  italicize_labels()
```

Comment:
Can we discuss a unified way of having objects outside teh code chunk to keep it organized across all the testing files?

Should we always put these types of objects at the top of the script? Something else?